src/adapter/src/coord.rs

```diff
@@ -1261,6 +1263,9 @@ pub struct Coordinator {
     /// Channel for strict serializable reads ready to commit.
     strict_serializable_reads_tx: mpsc::UnboundedSender<(ConnectionId, PendingReadTxn)>,
 
+    /// Channel for returning/releasing [ReadHolds](ReadHolds).
+    dropped_read_holds_tx: mpsc::UnboundedSender<ReadHolds<Timestamp>>,
```

Comment:
Can we just re-use `internal_cmd_tx` and send the drop messages over that?

Reply:
Yep, that seems like a good idea. I had first modeled it like the other special-purpose channels and it evolved from there... 😅

Reply:
I added a comment in the code to say why I think this might be good to keep:

```
    /// Channel for returning/releasing [InternalReadHolds](InternalReadHolds).
    ///
    /// We're using a special purpose channel rather than using
    /// `internal_cmd_tx` so that we can control the priority of working off
    /// dropped read holds. If we sent them as [Message] on the internal cmd
    /// channel, these would always get top priority, which is not necessary.
```

wdyt?

Reply:
> wdyt?

I think that's reasonable.